detectors/node/opentelemetry-resource-detector-aws/test/detectors/AwsBeanstalkDetector.test.ts

```diff
@@ -16,7 +16,7 @@
 
 import * as assert from 'assert';
 import * as sinon from 'sinon';
-import { awsBeanstalkDetector, AwsBeanstalkDetector } from '../../src';
+import { awsBeanstalkDetector, AwsBeanstalkDetectorSync } from '../../src';
```

Comment:
note for reviewer: `awsBeanstalkDetector` is delegating the job to `awsBeanstalkDetectorSync` so we need to apply the stubs into `AwsBeanstalkDetectorSync` class